src/components/StarRating/StarRating.scss

```diff
@@ -0,0 +1,31 @@
+$color-placeholder-star: #e1e1e1;
+$color-active-star: #d9af00;
+$color-number-reviews: gray;
+
+.CoveoStarRating {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
```

Comment:
I don't thinnnk you need these two lines. Both of those are the default attributes for `display:flex` iirc 🤔

Reply:
You are correct sir

Reply:
May be something you want to enforce though in the case where other CSS might be laying around in the page?
I would prefer being explicit here.